src/components/ccsds_command_depacketizer/gen/templates/assembly/name_ccsds_cosmos_plugin.txt

```diff
@@ -0,0 +1,12 @@
+# Set VARIABLEs here to allow variation in your plugin
+VARIABLE {{ name|lower }}_target_name {{ name|upper }}
+VARIABLE port_w 2003
+VARIABLE port_r 2003
```

Comment:
If this is the same port as hydra, we should change it so they are different. We don't want to prevent being able to run both in parallel.